src/schematools/contrib/django/models.py

```diff
@@ -260,6 +260,8 @@ def save_for_schema(self, schema: DatasetSchema):
         """Update this model with schema data"""
         self.schema_data = schema.json()
         self.auth = " ".join(schema.auth)
+        if new_path := schema.get("path"):
+            self.path = new_path
```

Comment:
```suggestion
        if new_path := schema.path:
            self.path = new_path
```

en dan een property `path` om het `DatasetSchema` object zetten. :-) Hebben we ook dat veld expliciet benoemd in de Python code i.p.v. overal raw de dict uitlezen.